src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.scss

```diff
@@ -6,14 +6,6 @@
     border-top: solid 1px $off-white;
     display: flex;
 
-    div[contentEditable="true"] {
```

Comment:
I think this was needed because of a CSS override somewhere in the elements code that does this
![jul-05-2018 13-07-38](https://user-images.githubusercontent.com/3299001/42345537-7d09fe48-8054-11e8-992e-dad6ca9824a1.gif)

Reply:
I think it's coming from the forms css that's imported from BRUI actually.